src/components/ProfileSolution/ProfileSolution.tsx

```diff
@@ -13,28 +10,34 @@ export interface Address {
   country: string;
 }
 
+export interface ProfileProps {
+  username: string;
+  showAddresses: boolean;
+}
+
+const Username = ({ username }: { username: string }) => {
+  const context = useContext(ColorContext);

```

Comment:
Juste histoire de rajouter de la déstructuration sur le contexte ?
```suggestion
  const { color } = useContext(ColorContext);
```
J'avais presque envie de proposer d'autres commentaires pour ajouter plusieurs couleurs, ça reste un peu overkill pour l'exercice..